Fix chrome alarms.get() return type

kstonekuan opened the pull request:
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developer.chrome.com/docs/extensions/reference/api/alarms#method-get>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

---

Both the documentation from chrome and my personal testing in chrome extensions show that the return type for `alarms.get()` can be `undefined`.

erwanjugand commented:
Hello @kstonekuan, thanks for your contribution. We should also update the type with the "callback" function (like `(alarm?: Alarm) => void`) and ideally add a test to confirm what is expected :)

kstonekuan commented:
Hey @erwanjugand, thanks for pointing that out. I've also fixed the callback type. 

Scope creep, but since I added the missing tests for `alarms.get()`, I just added the missing tests for all the `alarms` functions.

kstonekuan commented:
Ready to merge
